core/src/main/java/com/linecorp/armeria/server/VirtualHostServiceBindingBuilder.java

```diff
@@ -71,11 +71,21 @@ public VirtualHostServiceBindingBuilder path(String pathPattern) {
         return (VirtualHostServiceBindingBuilder) super.path(pathPattern);
     }
 
+    /**
+     * {@inheritDoc}
+     * @deprecated Use {@link #pathPrefix(String)}.
+     */
     @Override
+    @Deprecated
     public VirtualHostServiceBindingBuilder pathUnder(String prefix) {
         return (VirtualHostServiceBindingBuilder) super.pathUnder(prefix);
```

Comment:
ditto

Reply:
I migrated all methods which call `pathUnder`. Thanks for nice reviews!